Wrong Java classpath set in generic launcher POSIX shellscript for BlueJ 5.2.0

When installing BlueJ 5.2.0 using the generic installer ("BlueJ-generic-520.jar") on a UNIX-like operating system, it creates a small POSIX shellscript named "bluej" for launching the program.

In that file you specify a classpath that was still correct for BlueJ 5.1.0:

`CP="$APPBASE/lib/bluej.jar"`

However, this breaks since BlueJ 5.2.0 with the following error message:

```
Error: Could not find or load main class bluej.Boot
Caused by: java.lang.ClassNotFoundException: bluej.Boot
```

It seems the class was moved from bluej.jar to boot.jar, so the classpath set in the launcher needs to be changed accordingly:

`CP="$APPBASE/lib/boot.jar"`

After the change, BlueJ 5.2.0 works normally, as expected.

--- a/bluej/package/installer.props
+++ b/bluej/package/installer.props
@@ -32,14 +32,14 @@ install.javafx.classpath.unix = JAVAFX_CP="$JAVAFXPATH/lib/javafx.base.jar:$JAVA
 install.javafx.classpath.win = set JAVAFX_CP="%JAVAFXPATH%\\lib\\javafx.base.jar;%JAVAFXPATH%\\lib\\javafx.controls.jar;%JAVAFXPATH%\\lib\\javafx.fxml.jar;%JAVAFXPATH%\\lib\\javafx.graphics.jar;%JAVAFXPATH%\\lib\\javafx.media.jar;%JAVAFXPATH%\\lib\\javafx.properties.jar;%JAVAFXPATH%\\lib\\javafx.swing.jar;%JAVAFXPATH%\\lib\\javafx.web.jar"
 
 # additional commands to be added to MacOS script before execution
-install.commands.mac = CP="$APPBASE/lib/bluej.jar:/System/Library/Java"
+install.commands.mac = CP="$APPBASE/lib/boot.jar:/System/Library/Java"
 
 # additional commands to be added to Unix script before execution
-install.commands.unix = CP="$APPBASE/lib/bluej.jar"
+install.commands.unix = CP="$APPBASE/lib/boot.jar"
 
 # additional commands to be added to Windows batch file before execution
 # (windows should not have quotes surrounding CP string)
-install.commands.win = set CP=~\\lib\\bluej.jar
+install.commands.win = set CP=~\\lib\\boot.jar
 
 # java command-line options for unix (icluding MacOS)
 # (UNIX must have quotes around the $CP on the actual

